src/Sylius/Behat/Context/Ui/Admin/ManagingCountryContext.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Ui\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Page\Admin\Country\CreatePageInterface;
+use Sylius\Behat\Page\Admin\Crud\IndexPageInterface;
+use Sylius\Component\Core\Test\Services\SharedStorageInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+final class ManagingCountryContext implements Context
```

Comment:
This matches subsection in our ``features`` dir. I'd say we keep it for now and look at removing it later. Let's see what other Contexts we will need.